ipatests/test_integration/test_caless.py

```diff
@@ -266,6 +266,10 @@ def prepare_replica(self, _replica_number=0, replica=None, master=None,
         tasks.prepare_host(master)
         tasks.prepare_host(replica)
         for filename in set(files_to_copy):
+            if filename == 'does_not_exist':
```

Comment:
Hello,

I think that this is quite unsafe to hardcode such name.

Wouldn't be better to use:

```python
filepath = os.path.join(self.cert_dir, filename)
if not os.path.exists(filepath):
    logger.info("File '%s' not found, copying to server skipped", filepath)
    continue
```

Reply:
Sounds OK to me.